doc/api/inspector.md

```diff
@@ -136,8 +136,34 @@ with an error. [`session.connect()`] will need to be called to be able to send
 messages again. Reconnected session will lose all inspector state, such as
 enabled agents or configured breakpoints.
 
+## Example usage
+
+### CPU Profiler
+
+Apart from debugger, various V8 Profilers are available through the DevTools
```

Comment:
`Apart from debugger` -> `Apart from the debugger` perhaps?

Reply:
Thanks. Fixed.